.golangci.yml

```diff
@@ -246,3 +246,16 @@ linters:
     - exhaustive
     - gci 
     - noctx
+    - gofumpt
```

Comment:
@tanvigour one quick question are these the same set of linters excluded in dapr/dapr ? Apart from the ones introduced because of golangci version differences ?

Reply:
Thanks for checking on that Mukundan, I had a couple of older version ones I removed. The new commit has excludes only from newer versions.